apps/condo/pages/auth/change-password.tsx

```diff
@@ -184,19 +184,21 @@ const ChangePasswordPage: AuthPage = () => {
                             </Row>
                         </Col>
                         <Col span={24}>
-                            <Form.Item >
-                                <Button
-                                    key='submit'
-                                    type='sberPrimary'
-                                    loading={isSaving}
-                                    htmlType="submit"
-                                >
-                                    {SaveMsg}
-                                </Button>
-                                <Typography.Text type='secondary' style={{ marginLeft: '20px' }}>
-                                    {AndSignInMsg}
-                                </Typography.Text>
-                            </Form.Item>
+                            <Form style={{ maxWidth: '100%' }}>
```

Comment:
is it really needed here?

Reply:
Yes, otherwise I can not get to the property